### ISSUE TITLE
Use static thrusters

### ISSUE THREAD

**Issue by grimly** [1wsx10/VectorThrust2#8]
An idea I had while trying to solve #7 is to create 6 static nacelles each going in every direction and forcing them to output as much thrust as possible to lower the required force then dispatched between the nacelles.

A static "front" nacelle would then include :

* any thruster aiming to the front on the main body
* any thruster going in the normal direction of a nacelle which points forward
* any thruster going in the anti-normal direction of a nacelle which points backward

Basically, this reproduces the game strategy but can improve some design flaws of the current script like giving the ability to use low thrust on one dedicated axis

**Comment by 1wsx10** [comment]
what?

**Comment by grimly** [comment]
So far, your program do not allow a vanilla craft to fly. I think it can do it with low effort and it may solve some issues like what happen with the design I submitted in #7.

I have little extra time but I think I can manage to make a PR for that one.

**Comment by 1wsx10** [comment]
i dont know what you mean. you can make a pull request if you like

but it does work in vanilla

**Comment by 1wsx10** [comment]
im still not sure what you mean by this. closing due to lack of traffic
